### PR TITLE
vocabulary files

### PR DESCRIPTION
**Pull request by SofiaMp**

### CODE REVIEW

File: vocabulary/modified.csv [hunk 1]
```diff
@@ -0,0 +1,10 @@
+field_id,name,mapping_approach,date_field_id,modified_by_field_id,modified_by_value,target_concept_id,target_value_concept_id,target_unit_concept_id
+3894,Age heart attack diagnosed,Modified,53,6150,1,40770406,4329847,9448
+3627,Age angina diagnosed,Modified,53,6150,2,40770406,77670,9448
+4056,Age stroke diagnosed,Modified,53,6150,3,40770406,4310996,9448
+2966,Age high blood pressure diagnosed,Modified,53,6150,4,40770406,4086587,9448
+20551,Substance of prescription or over-the-counter medication addiction,Modified,53,20503,1,3184281,4156741,
+20551,Substance of prescription or over-the-counter medication addiction,Modified,53,20503,1,3184281,4189272,
+20551,Substance of prescription or over-the-counter medication addiction,Modified,53,20503,1,3184281,45878142,
+2986,Started insulin within one year diagnosis of diabetes,Modified,53,31/2443/4041,1/1,4188893,45884660,
+2986,Started insulin within one year diagnosis of diabetes,Modified,53,31/2443/4041,0/1/0,4188893,45884660,
```
Comment (MaximMoinat) [inline vocabulary/modified.csv:10]: How to interpret the 31/2443/4041 value for `modified_by_field_id`? Might be a good example for the lookup script.

File: vocabulary/modified.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+field_id,name,mapping_approach,date_field_id,modified_by_field_id,modified_by_value,target_concept_id,target_value_concept_id,target_unit_concept_id
+3894,Age heart attack diagnosed,Modified,53,6150,1,40770406,4329847,9448
+3627,Age angina diagnosed,Modified,53,6150,2,40770406,77670,9448
+4056,Age stroke diagnosed,Modified,53,6150,3,40770406,4310996,9448
+2966,Age high blood pressure diagnosed,Modified,53,6150,4,40770406,4086587,9448
```
Comment (MaximMoinat) [inline vocabulary/modified.csv:5]: Note that these are specifically _modified numeric_, where the value of the `field_id` goes into the `value_as_number` field. 

This is different from field _20551_, where the value maps to a `value_as_concept_id`.

So we need to separate out these mapping approaches.

File: vocabulary/do_not_capture.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+field_id,name,value,reason_std,reason,mapping_approach
```
Comment (MaximMoinat) [inline vocabulary/do_not_capture.csv:1]: There already exists the file [ignore.csv](https://github.com/thehyve/ukbiobank-omop-etl/blob/master/vocabulary/ignore.csv).

Please add the rows of _ignore.csv_ in _do_not_capture.csv.

File: vocabulary/modified.csv (hunk 2 above)
Comment (SofiaMp) [reply vocabulary/modified.csv:5]: The reason I put them together is that part of the filed name is used to fill the value_as_concept_id as well in the example. "age heart attack diagnosed" --> value_as_concept_id = 4329847 (myocardial infarction).

**Comment by SofiaMp** [comment]
Did not see you approved the changes so I committed the notebooks for the metadata and tables in the same pull request.

File: src/Python/example_omop_table.ipynb [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    "import json\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "outputs": [],
+   "source": [
+    "from src.Python.get_metadata import examples"
```
Comment (MaximMoinat) [inline src/Python/example_omop_table.ipynb:21]: This script has not been added to git

File: src/Python/omop_table_examples.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+                       domain + '_concept_id': str(row.iloc[0]['target_concept_id']),
+                       domain + '_date': 'get date from field ' + str(row.iloc[0]['date_field_id']) + '-' + instance,
+                       'value_as_concept_id': str(row.iloc[0]['target_value_concept_id']),
+                       'value_as_number': 'get value from field ' + str(row.iloc[0]['field_id']) + '-' + instance,
+                       'unit_concept_id': str(row.iloc[0]['unit_concept_id'])
+                       })
+    return table
+
+
+display(example_omop_table(2986, '0.1', 0))
```
Comment (MaximMoinat) [inline src/Python/omop_table_examples.py:34]: The instance id is only the number before the dot: `<field_id>.<instance_id 0-3>.<array_index 0-N>`

So the column name should be `field_id + '-' + instance + '.0'`

File: src/Python/omop_table_examples.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+def omop_domain(field):
+    concept_id = int(examples[examples['field_id'] == field]['target_concept_id'].unique())
+    response = requests.get(f'https://athena.ohdsi.org/api/v1/concepts/{concept_id}')
+    concept = json.loads(response.text)
+    domain = concept['domainId']
+    return domain
+
+
+def example_omop_table(field, instance, *value):
```
Comment (MaximMoinat) [inline src/Python/omop_table_examples.py:17]: Currently, the function can only process one value. Maybe later we can allow multiple values to be given. So replace `*value` with just `value` for now.

File: src/Python/omop_table_examples.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    domain = omop_domain(field)
+    if value:
+        row = examples[(examples['field_id'] == field) & (examples['value'] == value)]
+    else:
+        row = examples[examples['field_id'] == field]
+
+    table = pd.Series({'person_id': 'get(eid)',
```
Comment (MaximMoinat) [inline src/Python/omop_table_examples.py:24]: This does not take the mapping approach into account currently. 
e.g. for a numeric approach you would not have a value_as_concept_id.

**Comment by SofiaMp** [comment]
the display of the concept_ids' is a float (not looking the best) but if I make it int and it's not there I cannot drop the NaN anymore so In left it for now.

**Pull request merged**